enums.h

```diff
@@ -1173,9 +1173,9 @@ enum missile_id {
     MIS_FIREMOVE      = 0xE,
     MIS_CHAIN         = 0xF,
     MIS_NULL_10       = 0x10, // sentinal?
-    MIS_NULL_11       = 0x11,
-    MIS_NULL_12       = 0x12,
-    MIS_NULL_13       = 0x13,
+    MIS_NULL_11       = 0x11, // BLODSTAR

```

Comment:
Hey, any reason why these aren't just renamed?

Reply:
Wasn't to sure, some of them are never triggered. I'll rename them for now and we can correct them if we find more details at a later point?

Reply:
Yup, sounds good. 👍

Reply:
Did some further digging but didn't find any real info, but at least not missiledata is aligned and the missile_id enum is being used in more places :)